src/MongoDB.Analyzer/MongoDBDiagnosticAnalyzer.cs

```diff
@@ -86,4 +93,16 @@ private static string GetFilePath(SemanticModelAnalysisContext context)
             return "Unknown";
         }
     }
+
+    private static bool IsDriverVersionValid(MongoAnalysisContext context)
```

Comment:
Should it be named `IsDriverVersionSupported`?

Reply:
Changed.